src/DocumentPicker.Plugin/Shared/StreamProvider/StreamProviderImplementation.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System.IO;
+using System.Threading.Tasks;
+
+namespace DocumentConverter.Plugin.Shared.StreamProvider
+{
+    public class StreamProviderImplementation : ICustomStreamProvider
+    {
+        public async Task<Stream> OpenAsync(string filePath)
+        {
+            var stream = await Task.FromResult(File.Open(filePath,FileMode.Open));
```

Comment:
please do not `await Task.FromResult`...

can you please just create the result stream and then return 
`return Task.FromResult(stream)`

that waqy, you can also remove the `async` keyword!